mock_drivers/rosbag_mock_drivers/src/MockControllerDriver.cpp

```diff
@@ -30,7 +41,11 @@ uint8_t MockControllerDriver::getDriverStatus()
 
 void MockControllerDriver::vehicleCmdCallback(const autoware_msgs::VehicleCmd::ConstPtr& msg)
 {
-  robot_enabled_ = true;  // If a command was received set the robot enabled status to true
+  //robot_enabled_ = true;  // If a command was received set the robot enabled status to true
+  CommandData command_data;
```

Comment:
You're declaration of CommandData needs to appear before it is first you. So you need to move that higher in the file.